core/domain/platform_feature_services.py

```diff
@@ -321,3 +341,22 @@ def get_platform_parameter_value(
     context = _create_evaluation_context_for_server()
     param = registry.Registry.get_platform_parameter(parameter_name)
     return param.evaluate(context)
+
+
+def get_platform_parameter_schema(param_name: str) -> Dict[str, str]:
+    """Returns the schema for the platform parameter.
+
+    Args:
+        param_name: str. The name of the platform parameter.
+
+    Returns:
+        Dict[str, str]. The schema of the platform parameter according
+        to the data_type.
+    """
+    parameter = registry.Registry.get_platform_parameter(param_name)
+    if parameter.data_type == 'string':
+        return {'type': 'unicode'}
+    elif parameter.data_type == 'number':
+        return {'type': 'float'}
+    else:
+        return {'type': parameter.data_type}
```

Comment:
Does this actually work? I feel it's better to do it explicitly for all cases and raise an exception if an invalid/unrecognized data type is provided.

You could also do this more declaratively by creating a dict that maps data_type to type. Just make sure that when you read off the value that you do a copy.deepcopy() (so that your value doesn't get modified when the original dict gets modified).

Reply:
Done